### PR TITLE
correct typo into new typo

### PR DESCRIPTION
**Pull request by iamcev**
Dad, I found a problem in your html and I "corrected" it. Hope you like it.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,6 +4,6 @@
         <title>My Awesome Page</title>
     </head>
     <body>
-        <h1>Welcome to my awsome page!</h1>
+        <h1>Welcome to my awseeereeqsdome page!</h1>
```
Comment (msdotnetclr) [inline index.html]: Well, I prefer it to be really *correct*

**Review by msdotnetclr — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (msdotnetclr) [inline index.html]: Oops, I accidentally approved it. Let's try again

**Review by msdotnetclr — CHANGES_REQUESTED** [review]
Please correct the spelling

File: index.html (hunk 1 above)
Comment (iamcev) [reply index.html]: jdjjksk hepl kd meemee

**Review by msdotnetclr — APPROVED** [review]
Good!

**Pull request merged**